scripts/Muon_Analysis_2.py

```diff
@@ -110,9 +170,11 @@ def saveToProject():
 def loadFromProject(project):
     global muonGUI
     muonGUI = main()
-    muonGUI.loadFromContext(project)
+    muonGUI.dock_widget.loadFromProject(project)
+    # muonGUI.loadFromContext(project)
```

Comment:
why have you commented this out?

Reply:
I'm uncertain, the new GUI doesn't currently use this code. I have uncommented for now.